cores/esp8266/LwipIntfDev.h

```diff
@@ -272,22 +274,39 @@ boolean LwipIntfDev<RawDev>::begin(const uint8_t* macAddress, const uint16_t mtu
         }
     }
 
-    if (_intrPin < 0
-        && !schedule_recurrent_function_us(
-            [&]()
-            {
-                this->handlePackets();
-                return true;
-            },
-            100))
+    if (_intrPin < 0 && !_scheduled)
     {
-        netif_remove(&_netif);
-        return false;
+        if (schedule_recurrent_function_us(
+                [&]()
+                {
+                    this->handlePackets();
+                    return true;
```

Comment:
You might want to return `_scheduled` here, which would stop and remove the scheduled function when it is set to `false` in `::end()`.
Then maybe `_started` can be used in place of this new `_scheduled`.
(For sanity, the one or the other would have to be `true` before the call to `schedule_recurrent_function_us()`)

Reply:
I didn't know how to remove the scheduled function. I rework it.

Reply:
done. I like it